### PR TITLE
Show Shaderpack in F3

### PR DESCRIPTION
**Pull request by IMS212**
pretty self-explanatory, looks in-game as "[Iris] Shaderpack: \<pack name>".

### CODE REVIEW

File: src/main/java/net/coderbot/iris/mixin/MixinDebugHud.java [hunk 1]
```diff
@@ -21,14 +21,8 @@
...
         List<String> messages = cir.getReturnValue();
 
-        
-
         messages.add("");
         messages.add("[Iris] Shaderpack: " + Iris.getIrisConfig().getShaderPackName());
-    
-       
-       
-
```
Comment (ColdIce1605) [inline src/main/java/net/coderbot/iris/mixin/MixinDebugHud.java:26]: is this space needed?

**Review by ColdIce1605 — CHANGES_REQUESTED** [review]

File: src/main/java/net/coderbot/iris/mixin/MixinDebugHud.java (hunk 1 above)
Comment (IMS212) [reply src/main/java/net/coderbot/iris/mixin/MixinDebugHud.java:26]: No, I'll remove it, one second

Comment (IMS212) [reply src/main/java/net/coderbot/iris/mixin/MixinDebugHud.java]: Fixed, along with adding a newline

**Review by coderbot16 — APPROVED** [review]
Looks good to me, thanks!

**Pull request merged**